base/setup/lib/utils/vollist.c

```diff
@@ -0,0 +1,207 @@
+/*
+ * PROJECT:     ReactOS Setup Library
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Volume utilities and list functions
```

Comment:
in vollist.h you say the PURPOSE is `Volume list functions`, but here you say it would be `Volume utilities and list functions`. Haromize both, or avoid adding duplicated bloat!